easyDiffractionApp/Gui/Pages/Experiment/SideBarGroups/PeakAsymmetry.qml

```diff
@@ -0,0 +1,84 @@
+// SPDX-FileCopyrightText: 2022 easyDiffraction contributors <support@easydiffraction.org>
+// SPDX-License-Identifier: BSD-3-Clause
+// © 2021-2022 Contributors to the easyDiffraction project <https://github.com/easyScience/easyDiffractionApp>
+
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+
+import easyApp.Gui.Style 1.0 as EaStyle
+import easyApp.Gui.Elements 1.0 as EaElements
+import easyApp.Gui.Logic 1.0 as EaLogic
+
+import Gui.Globals 1.0 as ExGlobals
+
+
+Grid {
+    columns: 4
+    columnSpacing: EaStyle.Sizes.fontPixelSize
+
+    Column {
+        EaElements.Label {
+            enabled: false
+            text: qsTr("P1")
+        }
+
+        EaElements.Parameter {
+            width: inputFieldWidth()
+            units: ""
+            text: EaLogic.Utils.toFixed(ExGlobals.Constants.proxy.parameters.instrumentParametersAsObj.reflex_asymmetry_p1.value)
+            onEditingFinished: editParameterValue(ExGlobals.Constants.proxy.parameters.instrumentParametersAsObj.reflex_asymmetry_p1["@id"], text)
+        }
+    }
+
+    Column {
+        EaElements.Label {
+            enabled: false
+            text: qsTr("P2")
```

Comment:
Is this short name `P1`, `P2` etc. good enough? Shouldn't we have at least a tooltip?
On the other hand, Labels don't really have tooltips  so maybe it's too much.

Reply:
Also, we might want to add a tooltip or a note about the method used, since apparently there is more than one way to do this correction. This one seems to be the Berar and Baldinozzi correction (?)

Reply:
I agree. This should more clear. I suggest extending it after we switch to the CrysPyV3 interface and confirm that peak asymmetry actually works. I have created a draft issue for the EasyDiffraction project.